.travis.yml

```diff
@@ -11,14 +9,18 @@ env:
     # We purposefully leave asserts on by excluding -NDEBUG
     - CMAKE_CXX_FLAGS="-DCMAKE_CXX_FLAGS_RELEASE=-O3 -g"
     - USE_LOG4CPP=-DUSE_LOG4CPP=TRUE
+    - USE_CONAN=-DUSE_CONAN=ON
+    - BUILD_DEPS=-DBUILD_DEPS=ON
```

Comment:
Unfortunately, if we are going to maintain builds with and without conan, I think we should ensure that both builds actually work and add them to the matrix. This means more jobs and longer CI times.

Reply:
I also don't think that we need to maintain two build methods.
I wanted to simplify our former build method to a case in which we have troubles with Conan (as in the last week) and of course I wouldn't want to make the build process longer.